### PR TITLE
labels and contextual helps fix

### PR DESCRIPTION
**Pull request by paddeo**

### CODE REVIEW

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpwebdynamicform/src/main/java/com/agiletec/plugins/jpwebdynamicform/apsadmin/global-messages_it.properties [hunk 1]
```diff
@@ -27,11 +27,11 @@ title.jpwebdynamicformMessageManager.management=Tipi Messaggio
...
 
-jpwebdynamicform.menu.messages.help=TO be inserted menu.messages
-jpwebdynamicform.menu.configuration.help=TO be inserted menu.configuration
-jpwebdynamicform.menu.messageTypes.help=TO be inserted menu.messageTypes
-jpwebdynamicform.configuration.help=TO be inserted configuration
-title.messageManagement.details.help=TO be inserted message.details
-title.messageManagement.newAnswer.help=TO be inserted message.newAnswer
-title.messageManagement.trash.help=TO be inserted message.trash
-title.messagetype.configuration.help=TO be inserted message.configuration
+jpwebdynamicform.menu.messages.help=Il plugin WEB DYNAMIC FORM consente agli amministratori di creare e gestire form dinamici da pubblicare nelle pagine di una applicazione web. Dalla pagina Tipo Messaggio, gli amministratori possono creare tipi di form: la procedura \u00e8 la stessa per la creazione di tipi di contenuto e profili utente. Dalla pagina di Configurazione, gli amministratori possono configurare i diverse tipi di form. Dalla pagina Lista Messaggi gli amministratori possono accedere all'elenco dei messaggi inviati dagli utenti delle applicazioni web / portali.
```
Comment (nicpuddu) [inline plugins/entando-plugin-jpwebdynamicform/src/main/java/com/agiletec/plugins/jpwebdynamicform/apsadmin/global-messages_it.properties]: controllare typo e sistemare problema sugli apici

Comment (paddeo) [inline plugins/entando-plugin-jpwebdynamicform/src/main/java/com/agiletec/plugins/jpwebdynamicform/apsadmin/global-messages_it.properties]: done

**Comment by entando-service** [comment]
All is Well

**Pull request merged**